tools/schemacode/schemacode/tests/conftest.py

```diff
@@ -9,19 +9,45 @@
 
 lgr = logging.getLogger()
 
+BIDS_SELECTION = [
+    "asl003",
+    "eeg_cbm",
+    "hcp_example_bids",
+    "micr_SEM",
+    "micr_SPIM",
+    "pet001",
+    "pet003",
+    "qmri_tb1tfl",
```

Comment:
Can you add in-line comments explaining each of the choices? Something like `# asl, t1w, longitudinal` would probably be sufficient in most cases.

Reply:
There isn't really any particular reason, other than me trying to cover a wide range of modalities and avoiding some of the heavier datasets... I guess I could drop one of the `pet*` ones and add something else instead. Also I'll probably want to add the OME.ZARR example once it gets merged, as it also allows us to test whether directory pseudo-files are handled correctly.

Reply:
I don't have an issue with the `pet` datasets. 6 seconds is _really_ fast!

Still, having some idea of what parts of the spec each example dataset covers will help us identify any gaps in the list.

Reply:
> Also I'll probably want to add the OME.ZARR example once it gets merged

That can be done once this is addressed: https://github.com/bids-standard/bids-examples/pull/315#issuecomment-1183408044

Reply:
@tsalo ok, added comments, not entirely sure the inline ones really help that much, since it's mostly in the archive name, but they don't bother much either :)